mbq/client/contrib/permissions.py

```diff
@@ -100,16 +109,9 @@ def __init__(self, client: ServiceClient):
         parsed = urllib.parse.urlparse(self.client._api_url)
         self.client._api_url = f"{parsed.scheme}://{parsed.netloc}"
 
-    def fetch_permissions(
-        self, person_id: UUIDType, org_ref: UUIDType
-    ) -> FetchedPermissionsDoc:
-        logger.debug(f"Fetching permissions from OS Core: {person_id}, {org_ref}")
-
+    def _make_get_request(self, *args, **kwargs):
```

Comment:
I don't know how I feel about this name, but I felt compelled to abstract this to stop repeating the same exception handling.